helper-cli/src/main/kotlin/common/Utils.kt

```diff
@@ -698,3 +700,12 @@ internal fun RepositoryConfiguration.merge(
             ruleViolations = resolutions.ruleViolations.mergeRuleViolationResolutions(other.resolutions.ruleViolations)
         )
     )
+
+/**
+ * Return a copy of this [OrtResult] with the [Repository.config] with the content of the given
+ * [respositoryConfigurationFile].
+ */
+fun OrtResult.replaceConfig(respositoryConfigurationFile: File?): OrtResult =
+    respositoryConfigurationFile?.let {
+        replaceConfig(it.expandTilde().readValue())
```

Comment:
`expandTilde()` should not be needed here as that's already done by the `convert {}` calls for *clikt* options.

Reply:
Done. (Also moved the function a bit upwards in the file).